docs/releases.md

```diff
@@ -0,0 +1,39 @@
+# Releasing to NPM
+
+## Checklist
+
+Unlike other projects in Lockbox, this project releases new versions as needed by other projects.
+
+* All finished work is verified to work as expected and committed to `master`
+* Engineering ahd PI have voiced approval to release (e.g., via Slack **#lockbox** channel)
+
+## Instructions
+
+**NOTE:** these instructions assume:
+
+* All of the [checklist items](#checklist) are complete
+* You are an administrator of the project `lockbox-datastore`
+* Your local git working copy has a remote named `upstream` pointing to `git@github.com:mozilla-lockbox/lockbox-datastore.git`
+
+To generate the next release binary:
+
+1. Update "version" in `package.json` (and `package-lock.json`):
+    - We follow the [semver](https://semver.org/) syntax
+    - _Prerelease_ (prior to _Stable_) have a major version of "0" (e.g., "0.2.0")
+    - _Beta_ releases will be labeled with "-beta" and an optional number (e.g., "0.2.0-beta" or "0.2.1-beta3")
+    - _Stable_ releases will **not** be albeled, and follow semver starting with "1.0.0"
+2. Update `CHANGELOG.md`:
+    - The latest release as at the top, under a second-level header
+    - Each release includes the sub headings "What's New", "What's Fixed", and "Known Issues"
+    - Consult with Product Management for wording if needed.
+3. Commit and ultimately merge to `master` branch.
+4. Create a pull request on Github [comparing changes from the `master` branch against/to `production`][production-compare]
+5. Tag the latest commit on `production` branch with an annotated version and push the tag:
+    - `git tag -a -m "Release 0.3.0" 0.3.0`
+    - `git push upstream 0.3.0`
+    - Travis-CI will build and publish to [NPM]
+6. Send an announcement to the team (e.g., via Slack **#lockbox** channel)
+
+[production-compare]: https://github.com/mozilla-lockbox/lockbox-datastore/compare/production...master
+[NPM]: https://npmjs.com/linuxwolf/lockbox-datastore
```

Comment:
This 404s. Are we deployed to npm yet? Even https://www.npmjs.com/package/lockbox-datastore didn't seem to resolve to a known package.

It looks like we're currently pulling the dependency from GitHub (via **lockbox-extension**)
https://github.com/mozilla-lockbox/lockbox-extension/blob/9b03172cc97b5a2716bad939e1369bb80c62bcd0/package.json#L63

Nit: it should be "npm" lowercase, per their style guide.

Reply:
Once we do publish this to npm (which we should just randomly do whenever so we can update lockbox-extension's reference), we should make sure we have a few people that can deploy new versions. Notably @jimporter and @devinreams and whomever else we think should have npm publish permissions. That should protect us from vacations and reduce the bus factor.

Reply:
@pdehaan I definitely agree there needs to be more on the publish list, but npmjs doesn't make that allowance before publishing for the first time.  I'll need to coordinate with them on npm user accounts, and I have an extra account created that we can had-off to someone(s).

I should have added the comment to this PR that the first publication will be special since we're playing catch-up.

Reply:
👍 for the special first couple releases until we get everything streamlined.

Also, self-reminder to verify the npm published files after we do an npm publish (to verify that we don't publish unnecessary files; such as docs and test files, etc).

Reply:
We can create a @mozilla.com Google Group with multiple members. Something like lockbox-npm@mozilla.com if so inclined...

Reply:
If it really were just having an email address, we can use lockbox@mozilla.com.  But the email address must be associated with a user account.

To start, I've put together such an account, but tied to my @mozilla.com email address to get through registration and verification.  We can change it later.

Publication happens through a user account.  We can (and must) add other npm collaborators to the package for publish rights.